docs/en/ingest-management/release-notes/release-notes-8.5.asciidoc

```diff
@@ -22,6 +23,28 @@ Also see:
 * {kibana-ref}/release-notes.html[{kib} release notes]
 * {beats-ref}/release-notes.html[{beats} release notes]
 
+// begin 8.5.2 relnotes
+
+[[release-notes-8.5.2]]
+== {fleet} and {agent} 8.5.2
+
+Review important information about the {fleet} and {agent} 8.5.2 release.
+
+[discrete]
+[[bug-fixes-8.5.2]]
+=== Bug fixes
+
+{fleet}::
+* No bug fixes in this release
```

Comment:
We have a bugfix included in 8.5.2: https://github.com/elastic/kibana/pull/144899

This is a fix for a known issue in 8.5.1: https://github.com/elastic/kibana/issues//144880